WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -33,6 +34,7 @@ public static int Main(string[] args)
 		AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
 		TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
 		bool runGui = true;
+		bool runGuiMinimized = args.Any(arg => arg.Contains("--silent"));
```

Comment:
Define the magic sting as a constant at StartupHelper.cs
I would rename it to something like a command: 'startsilent'.

Reply:
Good point, but I want to make this work on every platform first.